eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -180,14 +179,6 @@ public final class AssembleMojo extends SafeMojo {
         defaultValue = "true")
     private boolean failOnError = true;
 
-    /**
-     * Parsed cache directory.
-     * @checkstyle MemberNameCheck (7 lines)
-     */
-    @Parameter(property = "eo.cache")
-    @SuppressWarnings("PMD.ImmutableField")
```

Comment:
@potatmen we cannot remove the property from `AssembleMojo` because it guarantees that it's passed into downstream mojos which are run within `exec()`. 
It should be left here.

Reply:
@mximp If I leave it here I get this 
`[ERROR] Errors: 
[ERROR]   AssembleMojoTest.assemblesNotFailWithFailOnErrorFlag:133 » NullPointer mavenSession may not be null
[ERROR]   AssembleMojoTest.assemblesTogether:74 » NullPointer mavenSession may not be null
[INFO] 
`

Reply:
@potatmen try to run the test in isolation. Maven session is not involved within the test - mojos are run manually, not by Maven. Need to find out why removal of the property affects working with `mavenSession`. Can you debug it?

Reply:
@mximp Honestly, I have no idea how to debug it, but I will try, later will let you know about progress.

Reply:
@mximp I checked all downstream `Mojos` : 
```
final Moja<?>[] mojas = {
            new Moja<>(ParseMojo.class),
            new Moja<>(OptimizeMojo.class),
            new Moja<>(DiscoverMojo.class),
            new Moja<>(PullMojo.class),
            new Moja<>(ResolveMojo.class),
            new Moja<>(MarkMojo.class),
            new Moja<>(PlaceMojo.class),
        };
```
They all extend `SafeMojo` therefore as well as `AssembleMojo`, therefore I don't get a point of creating `cache` attribute inside `AssembleMojo` is `SafeMojo` already has it.

Reply:
@potatmen You are correct. My fault. I didn't take into account that `AssembleMojo` extends `SafeMojo` as well.
In this case removal of the property shouldn't be a problem.

Reply:
> They all extend `SafeMojo`. `AssembleMojo` also extends it, therefore I don't get a point of creating `cache` attribute inside `AssembleMojo` if `SafeMojo` already has it.

I don't see any `cache` attribute in `SafeMojo`. Maybe you can leave this attribute here in `AssembleMojo` and add it to `OptimizedMojo`?